### ISSUE TITLE
Provide a way of retrieving an enum value by name

### ISSUE THREAD

**Issue by tiziano88** [dart-lang/dart-protobuf#50]
Similar to https://github.com/dart-lang/dart-protobuf/blob/master/lib/src/protobuf/protobuf_enum.dart#L43 , but by name rather than by value.

**Comment by skybrian** [comment]
I'm not quite sure what you're asking, but I think a workaround might be to loop over the .values list to find the one you want.

**Comment by tiziano88** [comment]
Yes that's exactly what I'm doing at the moment, I am just suggesting that a method be added that encapsulates this logic, and does not require iterating over the values each time. I am happy to give it a try at implementing this, feel free to assign the issue to me (it appears I can't).

**Comment by skybrian** [comment]
For some reason I can't assign it to you either. I think it might be limited to the Dart org?

But feel free to work on it. (Note that we need to be careful that any new symbols don't conflict with whatever enum values are in the .proto file.)
